Gemfile

```diff
@@ -1,10 +1,14 @@
+
 source "https://rubygems.org"
 
 # Declare your gem's dependencies in metasploit-credential.gemspec.
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
 gemspec
 
+gem 'metasploit-concern', git: 'https://github.com/jmartin-r7/metasploit-concern', branch: 'add-support-for-rails-7'
+gem 'metasploit_data_models', git: 'https://github.com/jmartin-r7/metasploit_data_models', branch: 'add-support-for-rails-7'
+
```

Comment:
```suggestion
```

To be removed when https://github.com/rapid7/metasploit-concern/pull/40 & https://github.com/rapid7/metasploit_data_models/pull/200 have landed.